.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: cpp
+dist: trusty
+os: linux
+cache: apt
+branches:
+  only:
+    - master
+matrix:
+  include:
+   - env: BUILD_TYPE=RelWithDebInfo
+     script:
+      - docker build -t verify-c-common -f verify-c-common.Dockerfile .
+      - docker run -t verify-c-common /bin/bash -c "cd build && ninja test"
```

Comment:
once everything is working, (but maybe even now?), need to do runs with different options.
Currently, we have the following combinations:

* normal : which is what is run above
* cex : with --cex switch
* vacuity: with --vac switch
* normal with yices: --horn-bmc-solver=smt-y2
* cex with yices: --horn-bmc-solver=smt-y2

Some of these might time out, so we will need to exclude tests that time out. This can be done using `ctest` command and its flags.

As this gets complicated, it is best to package this in a script and then call the script from the docker container.

Reply:
to do this, would I just add ctest --cex (or whatever flag) in the travis file or would I need to change the cmake instructions in the dockerfile?